js/glowingbear.js

```diff
@@ -746,6 +746,31 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
         }
     };
 
+    function nickFromBufferline(bufferline) {
```

Comment:
this should be using the existing models from models.js. Why are you duplicating this?